Controller/Admin/ModuleSupport.php

```diff
@@ -53,15 +52,17 @@ public function render()
         $sModuleId = $this->getEditObjectId();
         $sDefaultEmail = $this->getConfig()->getActiveShop()->oxshops__oxinfoemail->value;
 
-        $this->_aViewData += [
+        $this->setViewData($this->getViewData() + [
             'oxid' => $sModuleId,
             'contactEmail' => $this->_getModule()->getInfo('email'),
             'defaultEmail' => $sDefaultEmail,
             'isOurModule' => Helper::isThisModule($sModuleId),
-        ];
+        ]);
 
-        if (empty($this->_aViewData['fromEmail'])) {
-            $this->_aViewData['fromEmail'] = $sDefaultEmail;
+        if (empty($this->getViewData('fromEmail'))) {
```

Comment:
Uniform your
`$this->setViewData($this->getViewData() + [])` so the first array is the existing viewData and the second is the additional one. It accrues in the whole project please adjust. 
It is also the time to make a Helper method where you just send the added array as the other part of the code is always the same.

Reply:
Added a `addToViewData` helper method for this.